docs/community-clients/index.asciidoc

```diff
@@ -132,6 +132,8 @@ The following project appears to be abandoned:
 
 * https://github.com/mbuhot/eskotlin[ES Kotlin]:
   Elasticsearch Query DSL for kotlin based on the {client}/java-api/current/index.html[official Elasticsearch Java client].
+  
+* https://github.com/jillesvangurp/es-kotlin-wrapper-client[ES Kotlin Wrapper Client] Kotlin extension functions and abstractions for the {client}/java-api/current/index.html[official Elasticsearch Highlevel Client]. Aims to reduce the amount of boilerplate needed to do searches, bulk indexing and other common things users do with the client.
```

Comment:
super small nit: could you add a colon after the link and put the decription on the next line? This seems to be the formatting in most other places on this page and improves readability of the source file IMHO. Also line wrapping
at about  ~120 chars would be nice although it isn't strictly necessary.